compiler/frontend/spark/src/main/java/org/apache/nemo/compiler/frontend/spark/transform/HDFSTextFileTransform.java

```diff
@@ -36,9 +36,9 @@
  *
  * @param <I> input type.
  */
-public final class HDFSTextFileTransform<I> extends NoWatermarkEmitTransform<I, String> {
+public final class HDFSTextFileTransform<I> extends NoWatermarkEmitTransform<I, String> { //NOSONAR
   private final String path;
-  private Path fileName;
+  private Path fileName; //NOSONAR
   private List<I> elements;
```

Comment:
Please revert the change made on line 39, and make the variables declared on line 41 and 42 `transient` to remove the code smell. You can declare `private transient ...` to make the changes. Thanks!